core/modules/parser/QSMySqlListener.cc

```diff
@@ -1432,6 +1545,25 @@ class ComparisonOperatorAdapter :
 public:
     using AdapterT::AdapterT;
 
+    void checkContext() const override {
+        if (_ctx->getText() == "=") {
+            return;
+        } else if (_ctx->getText() == "<") {
+            return;
+        } else if (_ctx->getText() == ">") {
+            return;
+        } else if (_ctx->getText() == "<>") {
+            return;
+        } else if (_ctx->getText() == "!=") {
+            return;
+        } else if (_ctx->getText() == ">=") {
+            return;
+        } else if (_ctx->getText() == "<=") {
+            return;
+        }
```

Comment:
My personal approach to this kind of problems is to use the following pattern:
```
bool in(std::string const& what,
        std::vector<std::string>& const collection) {
   return collection.end() != std::find(collection.begin(), collection.end(), what);
}
```
Then you may write your method as:
```
if (in(_ctx->getText(),
       {"=", "!=", "<", "<=", ...})) return;
```
This utility function could be reusable many contexts.
I love functional programing :-)